src/device-registry/utils/create-event.js

```diff
@@ -1166,7 +1166,7 @@ const createEvent = {
           device_id: "6228c43567c2db20bffaa0cb",
           device_number: 0,
           device: "aq_613_97",
-          latitude: " 0.2857506",
+          latitude: "0.2857506",
           longitude: "32.5783253",
           pm2_5: 45.11,
           pm10: 39.16,
```

Comment:
Curious why we have hardcorded data @Baalmart

Reply:
This was just some test data.....for a forthcoming PR on consumption from Kafka. @NoahNsimbe